mpa/det/incr_stage.py

```diff
@@ -0,0 +1,282 @@
+# Copyright (C) 2022 Intel Corporation
```

Comment:
It would be okay to rename this to stage.py because we have seperated folder for semi.